.gitmodules

```diff
@@ -4,7 +4,7 @@
 [submodule "lib/eigenlayer-contracts"]
 	path = lib/eigenlayer-contracts
 	url = https://github.com/Layr-labs/eigenlayer-contracts
-	branch = dev
+	branch = feat/rewards-v2
```

Comment:
This will be changed and git submodule updated once the upstream changes in `eigenlayer-contracts` are merged to `dev`. Tracking in https://github.com/Layr-Labs/eigenlayer-contracts/pull/837

Reply:
Done.